### PR TITLE
#3572 Saving graph in manager pipeline

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3572 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Toolbox/Toolbox.js [hunk 1]
```diff
@@ -91,12 +96,17 @@ const Toolbox = props => {
   const [sources, sinks] = getConnectorInfo(currentWorker);
 
   const privateTopic = {
-    settings: { name: 'Pipeline Only' },
+    settings: {
+      name: 'Pipeline Only',
+      tags: { type: 'private', label: 'Pipeline Only' },
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Toolbox/Toolbox.js]: 👍

File: ohara-manager/client/src/components/Pipeline/Toolbox/toolboxUtils.js [hunk 2]
```diff
@@ -116,10 +116,10 @@ export const createToolboxList = params => {
       new joint.shapes.html.Element({
         position: { x: 10, y: index * 40 },
         size: { width: 272 - 8 * 2, height: 40 },
-        displayName: topic.displayName,
+        displayName: topic.type === 'private' ? topic.label : topic.displayName,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Toolbox/toolboxUtils.js]: So we're showing the private topic name in paper as well?

Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Toolbox/toolboxUtils.js]: ok

Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Toolbox/toolboxUtils.js]: thanks

**Pull request merged**